provider/virt_storage/storage_volume.py

```diff
@@ -191,6 +191,12 @@ def refresh_protocol_by_params(self, params):
         discard = params.get("image_discard_request", "unmap")
         self.protocol.set_param("auto-read-only", auto_readonly)
         self.protocol.set_param("discard", discard)
+        # image_aio:native requires cache.direct:on
+        if params.get("image_aio") == "native":
+            direct = params.get("image_cache_direct", "on")
+            no_flush = params.get("image_cache_no_flush", "off")
```

Comment:
Hi @aliang123 , what about removing these two lines if we don't need to do negative tests, we can set direct=on and no-flush=off directly. Then we don't need to introduce two more case-level params.

Reply:
@zhencliu , yes, I think we can remove them, because aio=native only allow the cache.direct=on. We don't need other negative test scenario at present, so will update it as zhenchao's suggestion. Thanks, zhenchao.